pallets/committee/src/benchmarking.rs

```diff
@@ -0,0 +1,216 @@
+                .generate_did()
+                .seed(i)
+                .build("member")
+        })
+        .collect();
+    Members::<I>::put(members.iter().map(|m| m.did()).collect::<Vec<_>>());
+    make_proposals_and_vote::<T, I>(members.as_slice())?;
+    Ok(members)
+}
+
+benchmarks_instance! {
+    _ {}
+
+    set_vote_threshold {
+        let n = 1;
+        let d = 2;
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_vote_threshold(n, d);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(Module::<T, _>::vote_threshold() == (n, d), "incorrect vote threshold");
+    }
+
+    set_release_coordinator {
+        let dids: Vec<_> = (0..COMMITTEE_MEMBERS_NUM)
+            .map(|i| UserBuilder::<T>::default()
+                 .generate_did()
+                 .seed(i as u32)
+                 .build("member")
+                 .did()
+            ).collect();
+        let coordinator = dids[COMMITTEE_MEMBERS_NUM / 2].clone();
+        Members::<I>::put(dids);
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_release_coordinator(coordinator);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(
+            Module::<T, _>::release_coordinator() == Some(coordinator),
+            "incorrect release coordinator"
+        );
+    }
+
+    set_expires_after {
+        let maybe_block = MaybeBlock::Some(1.into());
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_expires_after(maybe_block);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(Module::<T, _>::expires_after() == maybe_block, "incorrect expiration");
+    }
+
+    vote_or_propose_new_proposal {
+        let m in 2 .. COMMITTEE_MEMBERS_MAX;
+
+        let members = make_members_and_proposals::<T, I>(m)?;
+        let last_proposal_num = ProposalCount::<I>::get();
+        let proposal: <T as Trait<I>>::Proposal =
+            frame_system::Call::<T>::remark(vec![0; PROPOSAL_PADDING_LEN]).into();
+        let hash = <T as frame_system::Trait>::Hashing::hash_of(&proposal);
+        identity::CurrentDid::put(members[0].did());
+    }: vote_or_propose(members[0].origin.clone(), true, Box::new(proposal.clone()))
+    verify {
+        // The proposal was stored.
+        ensure!(Proposals::<T, I>::get().contains(&hash), "new proposal hash not found");
+        ensure!(ProposalOf::<T, I>::get(&hash) == Some(proposal), "new proposal not found");
+    }
+
+    vote_or_propose_existing_proposal {
+        let m in 2 .. COMMITTEE_MEMBERS_MAX;
+
+        let members = make_members_and_proposals::<T, I>(m)?;
+        let proposal: <T as Trait<I>>::Proposal =
+            frame_system::Call::<T>::remark(vec![1; PROPOSAL_PADDING_LEN]).into();
+        let hash = <T as frame_system::Trait>::Hashing::hash_of(&proposal);
```

Comment:
Used thrice, difference is only the number; let's extract this to a function returning a tuple.